frontend/src/container/AdvancedSearch.jsx

```diff
@@ -147,16 +125,22 @@ export class AdvancedSearch extends React.Component {
     const { props } = this
     // INFO - G.B. - 2021-02-12 - check if the user comes through an url that is not placed at first page
     const hasFirstPage = !(currentSearchLength < searchObject.numberResultsByPage * (searchObject.currentPage - 1))
+    const onlyGetFacet = Object.keys(appliedFilters).length === 0 && !searchObject.searchString
+
+    let pageNumber = FIRST_PAGE
+    let pageSize = searchObject.numberResultsByPage
+    if (onlyGetFacet) {
+      pageSize = 0
```

Comment:
It will be nice to have a comment to explain this 0

Reply:
Done.